src/clj/game/cards/resources.clj

```diff
@@ -1139,6 +1139,17 @@
                  :effect (effect (derez target)
                                  (trash card {:cause :ability-cost}))}]}
 
+   "Miss Bones"
+   {:data {:counter {:credit 12}}
+    :implementation "Credit use restriction not enforced"
+    :abilities [{:counter-cost [:credit 1]
+                 :msg "gain 1 [Credits] for trashing installed cards"
```

Comment:
Might want this to say somethig similar to what Scrubber says, if it doesnt already.

Reply:
Actually `Scrubber` doesn't mention the `trashing only` part. Probably should update it to add that to the log.

Reply:
That, or leave it as just "take 1 credit"